packages/shaders-react/src/index.ts

```diff
@@ -1,72 +1,56 @@
-// ----- ShaderMount ----- //
 export { ShaderMount } from './shader-mount';
 
-// ----- Fragment shaders ----- //
-
-// Mesh gradient
 export { MeshGradient, meshGradientPresets } from './shaders/mesh-gradient';
-export { type MeshGradientProps } from './shaders/mesh-gradient';
-export { type MeshGradientUniforms, type MeshGradientParams } from '@paper-design/shaders';
```

Comment:
We should have a single `export` statement per export path (except for `export type`, which gets removed anyway)

Otherwise esbuild preserves them and spits out multiple empty `import {} from './some-place'` in the build